components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1675,6 +1675,9 @@ void AdsServiceImpl::ShowScheduledCaptchaNotification(
     return;
   }
 
+#if BUILDFLAG(IS_ANDROID)
+  AdsServiceImpl::ShowScheduledCaptcha(payment_id, captcha_id);
+#else
```

Comment:
Can we abstract out the android and default implementations so that we do not need to use `#if BUILDFLAG`

Reply:
Agreed, that would make sense.